.circleci/config.yml

```diff
@@ -118,8 +118,15 @@ jobs:
       - run:
           command: |
               TEST=$(circleci tests glob **/__test*__/*.ts* | circleci tests split)
-              yarn test $TEST
+              mkdir -p ./reports/junit/
+              yarn test:ci $TEST --reporters=default --reporters=jest-junit
+          environment:
+            JEST_JUNIT_OUTPUT_DIR: ./reports/junit
       - run: ./scripts/codecov.sh
+
+      - store_test_results:
+          path: ./reports/junit/
```

Comment:
not entirely sure whether it applies here, but we are storing e2e artifacts under [/tmp/io-dev-api-server.log](https://github.com/pagopa/io-app/blob/master/.circleci/config.yml#L444) and [/tmp/e2e_artifacts/detox/](https://github.com/pagopa/io-app/blob/master/.circleci/config.yml#L454)

Do you think we can uniform this behavior?

Reply:
Sure, they are different things because in this step we don't save an artifact, but for sure it is better to use the `tmp` folder. Changed in 3f9c95f!